src/adhocracy_core/adhocracy_core/authorization/__init__.py

```diff
@@ -87,3 +87,26 @@ def get_local_roles_all(resource) -> dict:
                                            roles)
             local_roles_all[principal].update(roles_without_creator)
     return local_roles_all
+
+
+def acm_to_acl(acm: dict, registry: Registry) -> [str]:
+    """Convert an Access Control Matrix into a pyramid ACL.
+
+    To avoid generating too many ACE, action which are None will not
+    generate an ACE.
+
+    """
+    acl = []
+    idx = 0
+    for principal in acm['principals']:
+        for permissions in acm['permissions']:
+            permission_name = permissions[0]
+            if permission_name not in registry.content.permissions:
+                raise ValueError('Invalid permission: {0} not in {1}'
+                                 .format(permission_name, registry.content.permissions))
+            action = permissions[idx + 1]
+            if action is not None:
+                ace = (action, principal, permission_name)
+                acl.append(ace)
+        idx = idx + 1
```

Comment:
Is it really necessary as the override will be done with two ACMs and not inside the same ACM? For one ACM we don't have a Deny overriding a Allow for the same principal, permission couple.